CHANGELOG.md

```diff
@@ -1,52 +1,96 @@
-| Bug   | <!--PWA-2133-->Cart page out of stock crash                                             | [3447][]              |
-| Story | <!--PWA-2132-->Update Item Quantities                                                   | [3464][]              |
-| Bug   | <!--PWA-2131-->Improve Venia "create" scaffold script to stop relying on NPM log output | [3463][]              |
-| Story | <!--PWA-2004-->GQL Support for Contact Us & Newsletter                                  | `magento-commerce` PR |
-| Story | <!--PWA-1885-->Refactor Site Header to use Tailwind Theme                               | [3472][]              |
-
-## Bug fixes
-
--  [3464][] — **Cart**: Fixed an issue that prevented users from updating item quantities and removing items from the cart when using Magento 2.4-develop and 2.4.3 backends.
--  [3447][] — **Cart**: Fixed a rendering issue where out-of-stock products in your cart could not be removed.
--  [3495][] — **Cart**: Fixed issues that occurred when accessing the same store from different browser tabs or windows. To fix the problem, we introduced a storage listener on the cart context that fires a page reload whenever the `cartId` changes from another tab. Reloading the page forces Redux to persist a new `cartId` in storage.
--  [3535][] — **Cart**: Fixed a regression issue (during 12.1.0 development) that prevented the Add to Cart button from working on simple products featured on the Venia Home page.
--  [3513][] — **Mega Menu**: Fixed broken `Header` style when using the `develop` branch in a scaffolded project.
--  [3463][] — **Scaffolding CLI**: Fixed the Venia `_buildpack/create.js` `DEBUG_PROJECT_CREATION` test flag that broke when using NPM versions >=`7.23`.
+
+| Type  |Description                                                                                                                             |GitHub PR             |
+| :---- | :-------------------------------------------------------------------------------------------------------------------------------------- | :-------------------- |
+| Story | \[Docs] Shimmer and UPWARD Inlining                                                                                                     | [3548][]              |
+| Story | Add Cypress test coverage: Venia Filters, Search, Sort/Pagination                                                                       | [3519][][files][]     |
+| Story | Add Cypress test coverage: VeniaRemoveItemFromMiniCart                                                                                  | [3508][]              |
+| Story | Add Cypress test coverage: VeniaGuestIsAbleToAccessCategories.xml                                                                       | [3507][]              |
+| Story | Add Cypress test coverage: VeniaCartPage\*                                                                                              | [3509][][34][]        |
```

Comment:
```suggestion
| Story | Add Cypress test coverage: VeniaCartPage\*                                                                                              | [3509][], [34][]        |
```

Reply:
I won't comment on each one, but a comma between the PRs would be nice because it just gets rendered all next to each other:

![Screen Shot 2022-01-28 at 10 44 10 AM](https://user-images.githubusercontent.com/13182778/151587088-975bf962-698d-4b67-9eac-bf8ac92c8933.png)